applications/crossbar/priv/couchdb/schemas/temporal_rules.json

```diff
@@ -29,6 +29,12 @@
             "description": "Whether the rule is enabled",
             "type": "boolean"
         },
+        "end_date": {
+            "default": 0,
```

Comment:
Generally, we prefer that the absence of the value would mean no end date. Remove the default; `end_date` should only have a value if it is a proper, meaningful timestamp.